dozer-tests/src/e2e_tests/checker/client.rs

```diff
@@ -361,7 +361,7 @@ fn oapi_type_matches(oapi_type: &dozer_api::openapiv3::Type, field_type: FieldTy
                 true
             }
         }
-        (Array(array_type), FieldType::Binary | FieldType::Bson) => {
+        (Array(array_type), FieldType::Binary | FieldType::Json) => {
```

Comment:
This check needs to be consistent with oapi documentation generation

Reply:
ok!

Reply:
matched to check on the `SchemaKind`